Project.toml

```diff
@@ -5,8 +5,11 @@ version = "0.8.0"
 
 [deps]
 AtmosphericProfilesLibrary = "86bc3604-9858-485a-bdbe-831ec50de11d"
+ClimaComms = "3a4d1b5c-c61d-41fd-a00a-5873ba7a1b0d"
+ClimaCommsMPI = "5f86816e-8b66-43b2-912e-75384f99de49"
 ClimaCore = "d414da3d-4745-48bb-8d80-42e94e092884"
 ClimaCorePlots = "cf7c7e5a-b407-4c48-9047-11a94a308626"
+ClimaCoreTempestRemap = "d934ef94-cdd4-4710-83d6-720549644b70"
```

Comment:
We need the topographic information in cache and the data are lat/lon nc. The current way of doing it is to read the nc and [regrid](https://github.com/CliMA/ClimaAtmos.jl/pull/1233/files#diff-5998b0c81ebd351d5381e741e99924caff9c7f59fe61ebcde242fae51a710b40R64-R86) it onto the model grid in the cache set up, which sits in `src/`. Do you have suggestions on where else should this sit?

Reply:
It's probably better that we build whatever fields are needed in something like `examples/orographic_drag_helper.jl`, and pass that into the orographic drag cache function. Also, why does this data need to be remapped? I thought that remapping was only needed for post processing?

Reply:
Thanks! I'll work on it now. 

We will need topographic information on each model grid to compute the wave source but the topo info is usually on regular lat/lon grid. So we need a remap from `rll` to `cgll` here. On the other hand, for post-processing, we remap from `cgll` to `rll` so that we can better viz the results.